CreateTeamsalongwithAPPs/CreateTeamsAlongWithApps.ps1

```diff
@@ -0,0 +1,126 @@
+
```

Comment:
Please add Synopsis block, i.e. from this template for inline help. https://github.com/TeamsAdminSamples/PowerShell/blob/main/z_Templates/New-PowerShell.ps1